### DISCUSSION TITLE
Dynamic source data

### DISCUSSION THREAD

**Discussion by srgiola** [awslabs/performance-dashboard-on-aws#724] [General] [answered]
Hello dears developers.

I want to add dynamic data to a dashboard, this data is in an RDS. **What element in the architecture or in the source code should i replace / update to accomplish this goal?**.

Thanks in advance.

Sergio Lara

**Comment by trietlu** [comment, marked as answer]
Hello @srgiola ,

Thanks for your question.  We have a Data Ingest API you can use to upload datasets.  Every time you call the API to upload a new dataset, the dashboards using the dataset will be updated also

Here's some information about the [Data Ingest API](https://github.com/awslabs/performance-dashboard-on-aws/blob/mainline/docs/api.md)

**Reply by srgiola** [reply]
Thanks a lot @trietlu it solve it
